addons/base_automation/models/base_automation.py

```diff
@@ -53,7 +53,7 @@ class BaseAutomation(models.Model):
         'ir.model.fields', string='Trigger Date',
         compute='_compute_trg_date_id',
         readonly=False, store=True,
-        domain="[('model_id', '=', model_id), ('ttype', 'in', ('date', 'datetime'))]",
+        domain="[('model_id', '=', model_id), ('ttype', 'in', ('date', 'datetime')), ('name', '!=', 'write_date')]",
```

Comment:
This should be a python constraints, the domain on the field is only for preventing it being set from the front-end, but you may want to prevent people to set it via code, which is not prevented with the domain.